common.h

```diff
@@ -13,6 +13,9 @@
 // full onion address, WITHOUT newline
 #define ONION_LEN 62
 
+// How many times we loop before a reseed
+#define WORK_LOOP_COUNT 1<<24
```

Comment:
why so low?
anything less than `SIZE_MAX-8` is just a waste of time.

Reply:
Also I don't think it's good idea to separate limit from definition of variable which holds it (harder to analyze for overflow). Either `#define WORK_LOOP_TYPE size_t` or make it how it was.